docs/projects/ad4630_fmc/index.rst

```diff
@@ -240,7 +277,24 @@ The Software GPIO number is calculated as follows:
      - INOUT
      - 32
      - 86
-     
+   * - adaq42xx_pgia_mux[0]*
+     - INOUT
+     - 33
+     - 87
+   * - adaq42xx_pgia_mux[1]*
+     - INOUT
+     - 34
+     - 88
+   * - max17687_rst**
+     - INOUT
+     - 35
+     - 89
+
+.. admonition:: Legend
+   :class: note
+
+   -   ``*`` instantiated, but used for ADAQ4224 only
+   -   ``**`` instantiated, but used for ADAQ4224 with isolated power supply
```

Comment:
After line 297, a new empty line should be added, because this error is appearing when building the documentation `hdl/docs/projects/ad4630_fmc/index.rst:298: WARNING: Explicit markup ends without a blank line; unexpected unindent.`